service/src/main/java/org/cbioportal/service/util/ChromosomeCalculator.java

```diff
@@ -1,6 +1,7 @@
 package org.cbioportal.service.util;
 
 import org.cbioportal.model.Gene;
+import org.cbioportal.model.ReferenceGenomeGene;
```

Comment:
is this referenced anywhere here? if not then import statement should be removed

Reply:
done.